projects/ngx-intl-tel-input/package.json

```diff
@@ -1,13 +1,13 @@
 {
 	"name": "ngx-intl-tel-input",
-	"version": "3.1.1",
+	"version": "3.1.3",
 	"peerDependencies": {
-		"@angular/common": "8.x - 11.x",
-		"@angular/core": "8.x - 11.x",
-		"@angular/forms": "8.x - 11.x",
-		"google-libphonenumber": "^3.2.3",
-		"intl-tel-input": "^17.0.3",
-		"ngx-bootstrap": "^6.0.0"
+		"@angular/common": "^12.1.4",
```

Comment:
i assume there are way too many version changes here, more than are required to get this change to work?